rewrite-java-11/src/main/java/org/openrewrite/java/isolated/ReloadableJava11Parser.java

```diff
@@ -124,13 +124,13 @@ private ReloadableJava11Parser(
         Options.instance(context).put("should-stop.ifError", "GENERATE");
 
         LOMBOK:
-        if (classpath != null && classpath.stream().anyMatch(it -> it.toString().contains("lombok"))) {
+        if (classpath != null && classpath.stream().anyMatch(it -> it.toString().contains("lombok-1.18.37"))) {
```

Comment:
I'm wondering if the below ` || part.toString().contains("rewrite-java-lombok"` will trigger now that this specifically looks for `-1.18.37`. Any insights there?

Reply:
If Lombok itself isn't on the classpath, we can skip this entire code path.